### PR TITLE
Fix javadocs

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/model/ArmadilloEntityModel.mapping [hunk 1]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_iuifbloz net/minecraft/client/render/entity/model/ArmadilloEntityModel
-	COMMENT Represents the model of an {@linkplain ArmadilloEntity}.
+	COMMENT Represents the model of an {@linkplain net.minecraft.entity.passive.ArmadilloEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/ArmadilloEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of an {@linkplain net.minecraft.entity.passive.ArmadilloEntity ArmadilloEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/ArmorStandArmorEntityModel.mapping [hunk 2]
```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/unmapped/C_hakxxgsi net/minecraft/client/render/entity/model/ArmorStandArmorEntityModel
-	COMMENT Represents the armor model of an {@linkplain ArmorStandEntity}.
+	COMMENT Represents the armor model of an {@linkplain net.minecraft.entity.decoration.ArmorStandEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/ArmorStandArmorEntityModel.mapping]:
```suggestion
	COMMENT Represents the armor model of an {@linkplain net.minecraft.entity.decoration.ArmorStandEntity ArmorStandEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/ArmorStandEntityModel.mapping [hunk 3]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_xoloxthr net/minecraft/client/render/entity/model/ArmorStandEntityModel
-	COMMENT Represents the model of an {@linkplain ArmorStandEntity}.
+	COMMENT Represents the model of an {@linkplain net.minecraft.entity.decoration.ArmorStandEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/ArmorStandEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of an {@linkplain net.minecraft.entity.decoration.ArmorStandEntity ArmorStandEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/AxolotlEntityModel.mapping [hunk 4]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_mybxhars net/minecraft/client/render/entity/model/AxolotlEntityModel
-	COMMENT Represents the model of an {@linkplain AxolotlEntity}.
+	COMMENT Represents the model of an {@linkplain net.minecraft.entity.passive.AxolotlEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/AxolotlEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of an {@linkplain net.minecraft.entity.passive.AxolotlEntity AxolotlEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping [hunk 5]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_gknrfysf net/minecraft/client/render/entity/model/BatEntityModel
-	COMMENT Represents the model of a {@linkplain BatEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.BatEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.BatEntity BatEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/BeeEntityModel.mapping [hunk 6]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_jijccysy net/minecraft/client/render/entity/model/BeeEntityModel
-	COMMENT Represents the model of a {@linkplain BeeEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.BeeEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/BeeEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.BeeEntity BeeEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/BoatEntityModel.mapping [hunk 7]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_ctjgvdtk net/minecraft/client/render/entity/model/BoatEntityModel
-	COMMENT Represents the model of a {@linkplain BoatEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.vehicle.BoatEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/BoatEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.vehicle.BoatEntity BoatEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/CatEntityModel.mapping [hunk 8]
```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/unmapped/C_zuutbrxn net/minecraft/client/render/entity/model/CatEntityModel
-	COMMENT Represents the model of a {@linkplain CatEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.CatEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/CatEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.CatEntity CatEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/EntityModel.mapping [hunk 9]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_fouvqrqy net/minecraft/client/render/entity/model/EntityModel
-	COMMENT Represents the model of an {@linkplain Entity}.
+	COMMENT Represents the model of an {@linkplain net.minecraft.entity.Entity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/EntityModel.mapping]:
```suggestion
	COMMENT Represents the model of an {@linkplain net.minecraft.entity.Entity Entity}.
```

File: mappings/net/minecraft/client/render/entity/model/FoxEntityModel.mapping [hunk 10]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_ktkdvaak net/minecraft/client/render/entity/model/FoxEntityModel
-	COMMENT Represents the model of a {@linkplain FoxEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.FoxEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/FoxEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.FoxEntity FoxEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/GiantEntityModel.mapping [hunk 11]
```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/unmapped/C_lectivwm net/minecraft/client/render/entity/model/GiantEntityModel
-	COMMENT Represents the model of a {@linkplain GiantEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.mob.GiantEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/GiantEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.mob.GiantEntity GiantEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/GoatEntityModel.mapping [hunk 12]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_xuywnanw net/minecraft/client/render/entity/model/GoatEntityModel
-	COMMENT Represents the model of a {@linkplain GoatEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.GoatEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/GoatEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.GoatEntity GoatEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/OcelotEntityModel.mapping [hunk 13]
```diff
@@ -1,35 +1,2 @@
...
-	COMMENT </tr>
-	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#RIGHT_FRONT_LEG}</td><td>Root part</td><td>{@link #rightFrontLeg}</td>
-	COMMENT </tr>
-	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#LEFT_FRONT_LEG}</td><td>Root part</td><td>{@link #leftFrontLeg}</td>
-	COMMENT </tr>
-	COMMENT </table>
-	COMMENT </div>
+	COMMENT Represents the model of an {@linkplain net.minecraft.entity.passive.OcelotEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/OcelotEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of an {@linkplain net.minecraft.entity.passive.OcelotEntity OcelotEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/PiglinEntityModel.mapping [hunk 14]
```diff
@@ -1,60 +1,3 @@
...
-	COMMENT </tr>
-	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#LEFT_EAR}</td><td>{@value EntityModelPartNames#HEAD}</td><td>{@link #leftEar}</td>
-	COMMENT </tr>
-	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#RIGHT_EAR}</td><td>{@value EntityModelPartNames#HEAD}</td><td>{@link #rightEar}</td>
-	COMMENT </tr>
-	COMMENT </table>
-	COMMENT </div>
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.mob.PiglinEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/PiglinEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.mob.PiglinEntity PiglinEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/RabbitEntityModel.mapping [hunk 15]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_gjaljbcn net/minecraft/client/render/entity/model/RabbitEntityModel
-	COMMENT Represents the model of a {@linkplain RabbitEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.RabbitEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/RabbitEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.passive.RabbitEntity RabbitEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping [hunk 16]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_cfzbxjhf net/minecraft/client/render/entity/model/VexEntityModel
-	COMMENT Represents the model of a {@linkplain VexEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.mob.VexEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.mob.VexEntity VexEntity}.
```

File: mappings/net/minecraft/client/render/entity/model/WitherEntityModel.mapping [hunk 17]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_oocbnngt net/minecraft/client/render/entity/model/WitherEntityModel
-	COMMENT Represents the model of a {@linkplain WitherEntity}.
+	COMMENT Represents the model of a {@linkplain net.minecraft.entity.boss.WitherEntity}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/WitherEntityModel.mapping]:
```suggestion
	COMMENT Represents the model of a {@linkplain net.minecraft.entity.boss.WitherEntity WitherEntity}.
```

File: mappings/net/minecraft/entity/passive/WolfEntity.mapping [hunk 18]
```diff
@@ -27,10 +27,9 @@ CLASS net/minecraft/unmapped/C_xokssljs net/minecraft/entity/passive/WolfEntity
 	METHOD m_mbvjvzbc getFurWetBrightnessMultiplier (F)F
 		COMMENT Returns this wolf's brightness multiplier based on the fur wetness.
 		COMMENT <p>
-		COMMENT The brightness multiplier represents how much darker the wolf gets while its fur is wet. The multiplier changes (from 0.75 to 1.0 incrementally) when a wolf shakes.
+		COMMENT The brightness multiplier represents how much darker the wolf gets while its fur is wet. The multiplier changes (from {@code 0.75} to {@code 1.0} incrementally) when a wolf shakes.
 		COMMENT
-		COMMENT @return Brightness as a float value between 0.75 and 1.0.
-		COMMENT @see net.minecraft.client.render.entity.model.TintableAnimalModel#setColorMultiplier(int)
+		COMMENT @return Brightness as a float value between {@code 0.75} and {@code 1.0}.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/passive/WolfEntity.mapping]:
```suggestion
		COMMENT @return brightness as a float value between {@code 0.75} and {@code 1.0}.
```

File: mappings/net/minecraft/world/WorldEvents.mapping [hunk 19]
```diff
@@ -11,8 +11,8 @@ CLASS net/minecraft/unmapped/C_kkbhcwdu net/minecraft/world/WorldEvents
...
 	COMMENT synced by {@link net.minecraft.client.world.ClientWorld#syncWorldEvent(net.minecraft.entity.player.PlayerEntity, int, net.minecraft.util.math.BlockPos, int) ClientWorld#syncWorldEvent} and
 	COMMENT {@link net.minecraft.client.world.ClientWorld#syncGlobalEvent(int, net.minecraft.util.math.BlockPos, int) ClientWorld#syncGlobalEvent} (for regular and global events respectively), and
-	COMMENT finally processed by {@link net.minecraft.client.render.WorldRenderer#processWorldEvent(int, net.minecraft.util.math.BlockPos, int) WorldRenderer#processWorldEvent} and
-	COMMENT {@link net.minecraft.client.render.WorldRenderer#processGlobalEvent(int, net.minecraft.util.math.BlockPos, int) WorldRenderer#processGlobalEvent} (for regular and global events respectively).
+	COMMENT finally processed by {@link net.minecraft.client.world.WorldEventHandler#processWorldEvent(int, net.minecraft.util.math.BlockPos, int) WorldRenderer#processWorldEvent} and
+	COMMENT {@link net.minecraft.client.world.WorldEventHandler#processGlobalEvent(int, net.minecraft.util.math.BlockPos, int) WorldRenderer#processGlobalEvent} (for regular and global events respectively).
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/WorldEvents.mapping]:
```suggestion
	COMMENT {@link net.minecraft.client.world.WorldEventHandler#processGlobalEvent(int, net.minecraft.util.math.BlockPos, int) WorldRenderer#processGlobalEvent} (for regular and global events, respectively).
```

File: mappings/net/minecraft/entity/SpawnGroup.mapping [hunk 20]
```diff
@@ -28,7 +28,7 @@ CLASS net/minecraft/unmapped/C_ormqdxci net/minecraft/entity/SpawnGroup
 		COMMENT Returns {@code true} if this group is spawned as animals, or {@code false}
 		COMMENT if this group is spawned as monsters.
 		COMMENT
-		COMMENT @see net.minecraft.world.World#setMobSpawnOptions(boolean, boolean)
+		COMMENT @see net.minecraft.world.World#setMobSpawnOptions(boolean)
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/SpawnGroup.mapping]:
```suggestion
		COMMENT @see net.minecraft.world.World#setMobSpawnOptions(boolean) World#setMobSpawnOptions(boolean)
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]
I think it's more readable when the packages aren't displayed for links

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — APPROVED** [review]

**Pull request merged**